src/2018/transitioning/traits/impl-trait.md

```diff
@@ -1,6 +1,6 @@
 # impl Trait
 
-`impl Trait` is new syntax that currently works in function signatures.
+`impl Trait` is new way to specify unnamed but concrete types that implement a specific trait.
```

Comment:
Should this be `. . .is the new way to specify. . .`?

Reply:
yes please!